include/onnxruntime/core/common/common.h

```diff
@@ -219,6 +219,12 @@ inline long long TimeDiffMicroSeconds(TimePoint start_time, TimePoint end_time)
   return std::chrono::duration_cast<std::chrono::microseconds>(end_time - start_time).count();
 }
 
+#ifdef _WIN32
+#define IF_CONSTEXPR if constexpr
```

Comment:
I don't think any compiler needs it for the purposes used. It optimizes ways dead code anyway.

Reply:
This is for getting away a compiler warning: C4127